x-pack/plugins/security_solution/public/management/components/management_page_view.tsx

```diff
@@ -30,20 +30,20 @@ export const ManagementPageView = memo<Omit<PageViewProps, 'tabs'>>((options) =>
     }
     return [
       {
-        name: i18n.translate('xpack.securitySolution.managementTabs.endpoints', {
+        name: i18n.translate('xpack.securitySolution.managementTabs.hosts', {
```

Comment:
I see you created a `translations.ts` module under `management/common/` where both Host and Policies tab names are defined - should they be used here as well?